itest/src/edu/stanford/nlp/coref/hybrid/ChineseCorefBenchmarkSlowITest.java

```diff
@@ -1,24 +1,20 @@
 package edu.stanford.nlp.coref.hybrid;
 
-import java.io.BufferedReader;
-import java.io.File;
-import java.io.IOException;
-import java.io.PrintStream;
-import java.io.StringReader;
+import java.io.*;
```

Comment:
Please don't do this.  I think I'm allergic to import *

Reply:
Honestly, I am with what you say. But i found `.*` in various places in the code, so I changed the default Eclipse settings, since I don't want to be proposing so many changes.

I am reverting this, so all imports are individual.